pyproject.toml

```
@@ -7,7 +7,9 @@ module = "aiojobs"
 author = "Andrew Svetlov"
 author-email = "andrew.svetlov@gmail.com"
 home-page = "https://github.com/aio-libs/aiojobs"
-requires=[]
+requires=[
+    "async-timeout >= 4"
```

```suggestion
    "async-timeout >= 4.0.0"
```
Let's keep the full version